### DISCUSSION TITLE
Qucs-S v24.4.1 released

### DISCUSSION THREAD

**Discussion by ra3xdh** [ra3xdh/qucs_s#1062] [Announcements]
# Qucs-S 24.4.1

This release contains no new features, only bugfixes and translation update. Here is the summary of the changes. 

## Bugfixes

* Improved diagrams rendering speed #1042
* Fixed hardcoded version in LC Cauer filter #1039
* Fixed pulsed current source issues #1059
* Fixed show exit status if Ngspice crashes #1001

## Localization

* Russian translation update #1038

## Packaging

* Prepare for Flatpak package #51 Thanks @NN708

**Comment by agrunte** [comment]
Please add a note that Qucs cannot be installed on Windows 7. The installations ends succesfully, but the programm cannot be started. A following message appears: "The procedure entry point GetPackageFamilyName could not be located in the DLL kernel32.dll". Microsoft says that it happens if you try to run a program which is not compatible with Windows 7, because GetPackageFamilyName was introduced in Windows 8.

**Comment by ra3xdh** [comment]
> but the programm cannot be started. A following message appears: "The procedure entry point GetPackageFamilyName could not be located in the DLL kernel32.dll"

This cannot be fixed, because Qt6 developers has dropped Windows 7 support. Please upgrade the system, switch to Linux, or use the older releases with Qt5. I will add a note in the documentation.
